pkg/cloud/openstack/services/loadbalancer/service.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loadbalancer
+
+import (
+	"fmt"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/utils/openstack/clientconfig"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+const (
+	networkPrefix   string = "k8s"
```

Comment:
prefix has been changed to `k8s-clusterapi` here: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/424/files

Please also change it here

Reply:
I changed the prefix.